test/unit/user/permissions_test.rb

```diff
@@ -90,8 +121,9 @@ class User::PermissionsTest < ActiveSupport::TestCase
     user.admin_sections = [:services]
     refute user.has_access_to_all_services?
 
+    # updating admin section doesn't remove service permissions
     user.admin_sections = [:plans]
-    assert user.has_access_to_all_services?
+    refute user.has_access_to_all_services?
```

Comment:
This is a desired change that allows changing the sections, while keeping the current list of services.